docs/en/docs/data-operate/import/import-way/load-json-format.md

```diff
@@ -507,6 +507,48 @@ Import result:
 105 {"order1":["guangzhou"]} 7
 ````
 
+6. Import Array by json
+Since the Rapidjson handles decimal and largeint numbers which will cause precision problems, 
+we suggest you to use json string to import data to array<decimal> or array<largeint> column.
```

Comment:
```suggestion
we suggest you to use json string to import data to `array<decimal>` or `array<largeint>` column.
```

Please modify it, otherwise the `build extension` checks can not be passed

Reply:
done